docs/source/06_resources/01_faq.md

```diff
@@ -51,6 +33,13 @@ The primary differences to Bonobo ETL and Bubbles are related to the following f
  - **Automatic dependency resolution for pipelines**. The `Pipeline` module also maps out dependencies between nodes and displays the results of this in a sophisticated but easy to understand directed acyclic graph.
  - Extensibility
 
+## How is Kedro similar to other projects?
```

Comment:
The two reasons below are reasons why you would use Kedro. So the pipeline is minimal and you don't have to learn new things to use the Data Catalog.

Reply:
Agreed, but I thought it worth calling out these points as similarities because it makes the same point, but also reassures. 

I'll reinforce the points you mention, without marketing hype, so killing two birds...